erpnext/stock/doctype/shipment/shipment.py

```diff
@@ -93,6 +95,10 @@ def validate_weight(self):
 			if flt(parcel.weight) <= 0:
 				frappe.throw(_("Parcel weight cannot be 0"))
 
+	def calculate_total_weight(self):
+		weight_of_goods = sum(flt(parcel.weight) for parcel in self.shipment_parcel)
```

Comment:
```suggestion
		weight_of_goods = sum(flt(parcel.weight) * parcel.count for parcel in self.shipment_parcel)
```

I guess the weight is understood per parcel and should be multiplied by count, right?

Reply:
Thanks for pointing that out! I've fixed it.